.eslintrc.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
   ],
   extends: [
     "react-app", // Use the recommended rules from CRA.
-    "../eslint/common",
-    "../eslint/react",
+    "./eslint/common",
+    "./eslint/react",
```

Comment:
We can probably just move the contents of these files to the root eslint file now, right?

Reply:
There's a slightly different config for the cypress tests so we'd need to merge it with that and make the rules apply to the right directories. I'll file an issue.